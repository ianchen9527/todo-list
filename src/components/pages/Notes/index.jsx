import React from "react";
import styles from "./index.module.scss";
import NoteList from "../../../containers/NoteList";

const Notes = () => {
  return (
    <div className={styles["wrapper"]}>
      <div className={styles["increaser-wrapper"]} />
      <div className={styles["note-list-wrapper"]}>
        <NoteList />
      </div>
      <div className={styles["note-viewer-wrapper"]} />
    </div>
  );
};

export default Notes;
