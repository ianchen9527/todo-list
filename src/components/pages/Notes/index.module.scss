@import "../../../stylesheets/constants/colors.scss";

.wrapper {
  width: 100vw;
  height: 100vh;
  background-color: $main-color;
  overflow: hidden;
  display: flex;
  flex-flow: row wrap;

  .increaser-wrapper {
    width: 100vw;
    height: 60px;
    background: red;
  }

  .note-list-wrapper {
    width: 30vw;
    height: calc(100vh - 60px);
    background: green;
  }

  .note-viewer-wrapper {
    width: 70vw;
    height: calc(100vh - 60px);
    background: blue;
  }
}
