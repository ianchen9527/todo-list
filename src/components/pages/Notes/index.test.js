import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Notes from './index';

describe('Notes', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Notes />);
  });

  it('should render without crashing', () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
