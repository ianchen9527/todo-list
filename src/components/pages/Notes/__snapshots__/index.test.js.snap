// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render without crashing 1`] = `
<div
  className="wrapper"
>
  <div
    className="increaser-wrapper"
  />
  <div
    className="note-list-wrapper"
  >
    <Connect(withLifecycle(NoteList)) />
  </div>
  <div
    className="note-viewer-wrapper"
  />
</div>
`;

exports[`Notes should render without crashing 1`] = `
<div
  className="wrapper"
>
  <div
    className="increaser-wrapper"
  />
  <div
    className="note-list-wrapper"
  >
    <Connect(withLifecycle(NoteList)) />
  </div>
  <div
    className="note-viewer-wrapper"
  />
</div>
`;
